mne_bids/path.py

```diff
@@ -225,36 +225,39 @@ class BIDSPath(object):
     Examples
     --------
     >>> bids_path = BIDSPath(subject='test', session='two', task='mytask',
-                             suffix='ieeg', extension='.edf')
+    ...                      suffix='ieeg', extension='.edf')
     >>> print(bids_path.basename)
     sub-test_ses-two_task-mytask_ieeg.edf
     >>> bids_path
-    BIDSPath(root: None,
+    BIDSPath(
+    root: None
+    datatype: ieeg
     basename: sub-test_ses-two_task-mytask_ieeg.edf)
     >>> # copy and update multiple entities at once
     >>> new_bids_path = bids_path.copy().update(subject='test2',
-                                                session='one')
+    ...                                         session='one')
     >>> print(new_bids_path.basename)
     sub-test2_ses-one_task-mytask_ieeg.edf
     >>> # printing the BIDSPath will show relative path when
     >>> # root is not set
     >>> print(new_bids_path)
     sub-test2/ses-one/ieeg/sub-test2_ses-one_task-mytask_ieeg.edf
-    >>> new_bids_path.update(suffix='channels', extension='.tsv')
+    >>> new_bids_path = new_bids_path.update(suffix='channels',
+    ...                                      extension='.tsv')
     >>> # setting suffix without an identifiable datatype will
     >>> # result in a wildcard at the datatype directory level
```

Comment:
this probably needs to be overhauled, in my tests this was set to "ieeg", instead of being "*"

Reply:
done